packages/hooks/src/useCollectionQuery/__mocks__/queries.ts

```diff
@@ -0,0 +1,72 @@
+import gql from "graphql-tag";
+
+export const LIST_QUERY = gql`
```

Comment:
is this actually the `list query` or the `conversation query`? @andrewjdmccann

Reply:
This was a very intentional change. While this particular implementation is a Conversation query, the "interface" is that all we really need is for it to return a list. We don't care about anything else.